en/config/actuators.md

```diff
@@ -471,32 +471,32 @@ To assign an actuator:
 ### Motor Configuration
 
 :::note
-If using PWM or OneShot ESCs, you should first perform [ESC Calibration](../advanced_config/esc_calibration.md) (this topic also covers motor configuration).
+If using PWM or OneShot ESCs, you should first perform [ESC Calibration](../advanced_config/esc_calibration.md) (this topic also covers PWM specific motor configuration).
 
-[DShot](../peripherals/dshot.md) and [DroneCAN](../dronecan/escs.md)/Cyphal ESCs do not require this configuration.
+[DShot](../peripherals/dshot.md) ESCs do not require configuration of the command limits but only rotation direction.
```

Comment:
@MaEtUgR So we still need to configure max/min/dicarmed etc for CAN ESCs?

Reply:
Yes, because in the cases I've seen a rather [generalized motor command message](https://github.com/dronecan/DSDL/blob/6537918298be25eb285dff3428604e078b00f9a9/uavcan/equipment/actuator/Command.uavcan) is used and each ESC model/configuration can map it differently. That's also why the option to configure the limit commands exists e.g. for DroneCAN: https://github.com/PX4/PX4-Autopilot/blob/39e04d971229f3ca0d1e080d55bf7ac2f7894562/src/drivers/uavcan/module.yaml#L15-L25